flake.nix

```diff
@@ -0,0 +1,91 @@
+{
+  description = "categorifier";
+  inputs = {
+    nixpkgs.url = "github:NixOS/nixpkgs/nixos-21.11";
+    flake-utils.url = "github:numtide/flake-utils";
+    concat = {
+      url = "github:con-kitty/concat/wavewave-flake";
+      inputs.nixpkgs.follows = "nixpkgs";
+      inputs.flake-utlis.follows = "flake-utils";
+    };
+  };
+  outputs = { self, nixpkgs, flake-utils, concat }:
+    flake-utils.lib.eachSystem [ "x86_64-linux" "x86_64-darwin" ] (system:
+      let
+        parseCabalProject = import ./parse-cabal-project.nix;
+        excluded = [
+          "categorifier-vec-integration"
+          "categorifier-vec-integration-test"
```

Comment:
Why are these excluded?

Reply:
huh. yeah. I now found that even those packages were not existent in this branch. :man_shrugging:  Thanks.